src/promptflow/promptflow/_sdk/_errors.py

```diff
@@ -68,14 +68,16 @@ class GenerateFlowToolsJsonError(PromptflowException):
 class BulkRunException(PromptflowException):
     """Exception raised when bulk run failed."""
 
-    def __init__(self, *, message="", failed_lines: int, total_lines: int, line_errors, module: str = None, **kwargs):
+    def __init__(self, *, message="", failed_lines, total_lines, line_errors, module: str = None, **kwargs):
```

Comment:
Why do we remove this type hint here?

Reply:
Because it maybe a string.
![image](https://github.com/microsoft/promptflow/assets/111329184/1bb8fac6-9a05-453d-bd25-7fb81dbf2cb0)